test/exchange/dfs-sell.js

```diff
@@ -8,6 +8,6 @@ describe('Dfs-Sell', function () {
 
     it('... should check best price from DFSPrices contract', async () => {
         await dfsSellTest();
-        dfsSellSameAssetTest();
+        // sdfsSellSameAssetTest();
```

Comment:
Can we maybe split up the dfs sell tests, so we have kyber/0x/onchain etc... to be ran separately so we don't have commented out logic in the tests?